src/Colors.js

```diff
@@ -30,9 +30,33 @@ const COLORS = {
 
 const colorValues = Object.values(COLORS);
 
-const randomColor = function() {
-    const colorIndex = Math.floor(Math.random() * colorValues.length);    
-    return colorValues[colorIndex];
+const randomColor = function(exclusionList=[]) {
+    const filteredList = exclusionList.length ? filterList(colorValues, exclusionList) : colorValues;
+    let colorIndex = -1;
+    while (colorIndex < 0 ) {
```

Comment:
This loop is kind of confusing. IMO we should just raise an exception if the filtered list ends up being empty. At that point we can just return `colorValues[colorIndex]` like we originally did

Reply:
I agree about most of this, I think instead of returning a random color we should return 1 set color (in this case I set it to be white)